src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml

```diff
@@ -42,7 +42,7 @@
 
         <StackPanel x:Name="VerbosityPanel" Grid.Row="2" Grid.ColumnSpan="2">
 
-            <TextBlock Text="SonarLint output verbosity for the analysis of additional languages:"
+            <TextBlock Text="Logs output verbosity:"
```

Comment:
`Logs output verbosity` sounds slightly odd.

`Log output verbosity`, `Logger output verbosity`, or `Log verbosity` perhaps. VS has:
![image](https://user-images.githubusercontent.com/10757559/206743010-c666863c-459b-481e-a200-437e4db9a392.png)

Or avoid the problem with something like `Logging level:`